Ports/poppler/package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+export VERSION_DATA='0.4.12'
```

Comment:
Do you need to export this variable? Otherwise, you can just include it as if it's another property (i.e. `version_data='0.4.12'`

Reply:
It did not allow me to add a property it does not know, so I choose that way.

Reply:
What is "It"?

We frequently add non-standard properties like `commit_hash`, and as far as I know, that is allowed?

Reply:
I will try again later, but I added a `version_data` property and got an error running the `package.sh` that said something like "`version_data` is not a function"